### PR TITLE
extension actions

### LINKED ISSUE #2224: extensions are missing browser/page action icons
Some people have difficulty figuring out how to use the password manager extensions because we don't display the browser action icons

cc @bradleyrichter

### PR DESCRIPTION
**Pull request by bridiver**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Test Plan:
Select Dashlane as password manager
Dashlane icon should appear next to Brave shields icon
Clicking Dashlane icon should open the Dashlane popup
Same thing for Lastpass

Fix #2224

### CODE REVIEW

**Comment by bridiver** [comment]
Still has a few small open todos

**Comment by bsclifton** [comment]
Great job with the tests on this one!  This should make adding new extensions really easy :smile:

**Comment by bridiver** [comment]
requires https://github.com/brave/electron/commit/8d8dc9d9c27cdfa504020c3adbd1adccc0070a0b

File: app/sessionStore.js [hunk 1]
```diff
@@ -289,6 +289,8 @@ module.exports.cleanAppData = (data, isShutdown) => {
   // if (data.tabs) {
   //   data.tabs = data.tabs.map((tab) => tabState.getPersistentTabState(tab).toJS())
   // }
+  // TODO(bridiver) - really just need to clear out tabId related stuff
+  delete data.extensions
```
Comment (bbondy) [inline app/sessionStore.js]: I'll change this when I PR my stuff.

**Pull request merged**